src/main/java/io/lakefs/PathMapper.java

```diff
@@ -0,0 +1,311 @@
+package io.lakefs;
```

Comment:
Let's move to a sub-package.  `io.lakefs` is too flat.

Reply:
done